README.md

```diff
@@ -80,7 +85,7 @@ Godel: Austrian logician and mathematician http://en.wikipedia.org/wiki/Kurt_G%C
   ```
   rosrun godel_network irb2400_client.sh [server ip] [robot namespace] true
   ```
-  Note:Robot namespce is the identity of individual robot, can't be the same.
+  Note:Robot namespace is the identity of individual robot, they can't be the same.
```

Comment:
How about I change to:
To support multiple Robots, Service can run multiple programs by giving different namespaces, so these namespaces can't be the same.